examples/plot_simulate_gamma.py

```diff
@@ -108,6 +108,21 @@
 from hnn_core.viz import plot_psd
 plot_psd(dpls[trial_idx], fmin=20., fmax=100., tmin=tmin)
 
+###############################################################################
+# Finally, we demonstrate the mechanistic link between PING and the GABAA decay
+# time constant (`tau2`). Using the same network/drive configuration as before,
+# we decrease `tau2` from 5 to 2 ms. This will shorten the effective
+# refactory period between L5 pyramidal cell spikes and increase the PING
+# frequency from ~50 to ~65 Hz.
+net.cell_types['L5_pyramidal'].p_syn['gabaa']['tau2'] = 2
```

Comment:
@rythorpe what do you think about adding a few print statements (perhaps in another code block) to show off how you can inspect the biophysical parameter values?

Reply:
You're thinking like a more verbose `Cell.__repr__()` method?

Reply:
Yeah exactly! I'm getting of the mindset that any API we add to modify parameters will need to come with a really straightforward way to inspect those parameters.

Reply:
That sounds great actually. I'll leave that for the next PR since there's talk of potentially combining the `Cell` and `CellResponse` classes.